tests/test_confint.py

```diff
@@ -47,6 +50,7 @@ def test_confint():
     assert np.all(confint1 != confint3)
 
 
+@pytest.mark.skipif(sys.version_info >= (3, 12), reason="requires python3.11 or lower.")
```

Comment:
What if you import `doubleml` inside the test `test_against_doubleml` so that it does not get imported globally? I think the error is even at import time

Reply:
It worked 🎉!

Reply:
Yes! Was just about to comment 😀 thanks for your help! Now I need to work towards acceptance of the stargazer pr. Do you have any feedback on the content of the new notebooks? Both of them might need some polish here and there.

Reply:
I manly looked into the CI issue. I can take a look into the notebooks themselves either today or tomorrow :)